.github/workflows/publish-package-docs.yaml

```diff
@@ -94,3 +94,10 @@ jobs:
           pushd build-directory/docs/docsite
           ln -s ${VERSION} _build/latest
           popd
+
+      - name: Deploy docs build
+        run: |
+          rm -rf /var/www/html/ansible-package-docs
+          pushd build-directory/docs/docsite/_build/html
+          rsync -avz . /var/www/html/ansible-package-docs
```

Comment:
FWIW, I'd replace this with an existing action from the list: https://github.com/marketplace?type=actions&query=rsync.